doc/user_guide/checkers/extensions.rst

```diff
@@ -715,3 +715,19 @@ While Used checker Messages
 :while-used (W0149): *Used `while` loop*
   Unbounded `while` loops can often be rewritten as bounded `for` loops.
   Exceptions can be made for cases such as event loops, listeners, etc.
+
+
+.. _pylint.extensions.consider_refactoring_into_while_condition:
+
+Consider refactoring into while condition checker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This checker is provided by ``pylint.extensions.consider_refactoring_into_while_condition``.
+Verbatim name of the checker is ``consider_refactoring_into_while_condition``.
+
+Consider refactoring into while checker Messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+:consider-refactoring-into-while-condition (R3501): *Consider using 'while %s' instead of 'while True:' an 'if', and a 'break'*
```

Comment:
This need to be regenerated with latest code using the doc generation command.

Reply:
done